### PR TITLE
support pass-header option in apache::fastcgi::server

### PR DESCRIPTION
**Pull request by janschumann**

### CODE REVIEW

File: templates/fastcgi/server.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
-FastCGIExternalServer <%= @faux_path %> -idle-timeout <%= @timeout %> <%= if @flush then '-flush' end %> -host <%= @host %>
+FastCGIExternalServer <%= @faux_path %> -idle-timeout <%= @timeout %> <%= if @flush then '-flush' end %> -host <%= @host -%>
+<%- if @pass_header -%> -pass-header <%= @pass_header %><% end %>
```
Comment (hunner) [inline templates/fastcgi/server.erb]: Could you actually change these two `-%>\n<%-` to just `%> <%` so they're on the same line? It seems a little confusing to have them be on two lines if you want the output to be a single line, and flush is already setting precedent for single-line code. Thanks!

**Comment by janschumann** [comment]
Yes it is a bin confusing. But isn´t the readability of the template also a criteria?

Please see #1368 .. This line might get even longer.

Thoughts?

**Comment by hunner** [comment]
Ah yeah, if there are lots of options to pass then the other option is to pre-set all the variables in a `<% %>` block before the line to the full string like
```erb
<%
  timeout = " -idle-timeout #{@timeout}"
  host = " -host #{@host}"
  if @pass_header and ! @pass_header.empty?
    pass_header = " -pass-header #{@pass_header}"
  else
    pass_header = nil
  end
  #...
  options =  timeout + flush + host + pass_header
%>
FastCGIExternalServer <%= @faux_path %><%= options %>`
```

**Comment by janschumann** [comment]
agreed. i´ll do that.

**Comment by hunner** [comment]
Sorry for the late response on this. Could you document your new parameter in the readme as well?

**Comment by janschumann** [comment]
Sure. 

While looking into the documentation of [mod_fastcgi](http://www.fastcgi.com/mod_fastcgi/docs/mod_fastcgi.html), I realized that there are some more options available, such as `appConnTimeout` and `user` etc.. Do you want me to implement them as well? In another pull-request?

**Comment by hunner** [comment]
It's okay as-is; I always say to write for future extensibility, but present simplicity :).

:+1:

**Pull request merged**